oauth2client/client.py

```diff
@@ -771,11 +771,11 @@ def _do_refresh_request(self, http):
         headers = self._generate_refresh_request_headers()
 
         logger.info('Refreshing access_token')
-        resp, content = transport.request(
+        response = transport._request(
```

Comment:
My rule? Maybe you mean @nathanielmanistaatgoogle's rule. I'm not sure what you're referring to here.